Project-I-Social-Card/social-card/src/components/CardComponents/CardContainer.js

```diff
@@ -1,2 +1,19 @@
-import React from 'react';
-import './Card.css';
+import React from "react";
+
+import "./Card.css";
+
```

Comment:
the card.css only needs to be imported into one component (the top-level of the card)